test/extended/etcd/etcd_backup_noconfig.go

```diff
@@ -0,0 +1,120 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	configv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kube-openapi/pkg/util/sets"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultBackupCRName       = "default"
+	testSchedule              = "*/5 * * * *"
+	testTimeZone              = "UTC"
+	testRetentionType         = configv1alpha1.RetentionTypeNumber
+	testRetentionNumber       = 3
+	backupServerContainerName = "etcd-backup-server"
+)
+
+var _ = g.Describe("[sig-etcd][OCPFeatureGate:EtcdBackupNoConfig][Serial] etcd", func() {
```

Comment:
```suggestion
var _ = g.Describe("[sig-etcd][OCPFeatureGate:AutomatedEtcdBackup][Serial] etcd", func() {
```

https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/backuphelpers/backuphelpers.go#L8C45-L8C64